docker/Dockerfile

```diff
@@ -45,6 +45,14 @@ RUN git clone -b 3.13 --single-branch --recursive -j"$(grep ^processor /proc/cpu
 # Numpy, Cython and Pillow nightly builds
 RUN python3 -m pip install -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy cython pillow
 
+# PyTorch nightly build
+RUN SUFFIX="$(echo "${CUDA_VERSION//.}" | head -c 3)" && \
+    if [[ "$SUFFIX" =~ ^(118|121|124)$ ]]; then \
+        python3 -m pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu"$SUFFIX"; \
+    else \
+        echo "No available PyTorch wheels for CUDA version $CUDA_VERSION"; \
+    fi
```

Comment:
I'm wondering about a scenario, where user puts incompatible version of CUDA and learns that here. Changing CUDA version rebuilds earlier stages. Do you think it would be worth to put the "warning" in earlier layer? On the other hand it complicated the Dockerfile...

Reply:
Even if we put it in an earlier layer messages during docker build are very easy to miss, especially here when there are a lot of compilation logs. I don't think we should count on that to notify users about this. If we really want users to be aware of that a better place would probably be the README.